<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Debug</class>
 <widget class="QWidget" name="Debug">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Debug</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Debug Title</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QListWidget" name="debugList">
        <item>
         <property name="text">
          <string>Output 1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Output 2</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
